build.gradle

```
@@ -10,6 +10,13 @@ archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
+repositories {
+    // Add repositories to retrive artifacts from in here.
+    // Loom adds the Fabric maven automatically so you don't
```

```suggestion
    // Loom adds Maven Central and the Fabric maven automatically so you don't
```
(also technically the mojang maven but users don't have to care about that)

---

if you really pick on that you should mention the Minecraft repo as well

---

How about "essential maven repositories" /s

---

```suggestion
    // Loom adds some essential maven repositories so you don't
```

---

I'd rather say you only use this when you need to depend on other mods